applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -252,10 +266,21 @@ def InitializeSolutionStep(self):
 
             # Update the DENSITY and DYNAMIC_VISCOSITY values according to the new level-set
             self._SetNodalProperties()
-
+          
             # Initialize the solver current step
             self._GetSolutionStrategy().InitializeSolutionStep()
 
+            # Accumulative Water volume error ratio due to level Set .Adding source term 
+            self.water_volume_after_transport=KratosCFD.FluidAuxiliaryUtilities.CalculateFluidNegativeVolume(self.GetComputingModelPart())
+
+            volume_error = (self.water_volume_after_transport -  self.system_volume)/ self.system_volume
+            self.volume_error_to_check_before_adding_source_term= self.water_volume_after_transport - self.system_volume
```

Comment:
```suggestion
            water_volume_after_transport = KratosCFD.FluidAuxiliaryUtilities.CalculateFluidNegativeVolume(self.GetComputingModelPart())

            volume_error = (water_volume_after_transport - system_volume) / system_volume
```
You don't require these member variables. Besides, I think that the ' self.volume_error_to_check_before_adding_source_term` is no longer used (maybe I'm wrong).